src/main/java/com/github/manolo8/darkbot/extensions/features/FeatureDefinition.java

```diff
@@ -162,4 +163,26 @@ public boolean canLoad() {
     public void addIssue(@NotNull String message, @NotNull String description, @NotNull Level level) {
         issues.add(message, description, PluginIssue.Level.values()[level.ordinal()]);
     }
+
+    /**
+     * @return true if exception cause was {@link FeatureException}
```

Comment:
I think this makes no sense.

This addIssue simply should add it normally, like any other issue being added, with the particularity that it will indeed handle feature exception in a different way when creating the error message, which is an implementation detail